adafruit_tsl2591.py

```diff
@@ -69,6 +69,8 @@
 _TSL2591_LUX_COEFB           = 1.64
 _TSL2591_LUX_COEFC           = 0.59
 _TSL2591_LUX_COEFD           = 0.86
+_TSL2591_MAX_COUNT_100MS     = const(36863)
```

Comment:
Please rewrite this number as 0x8fff to match the one below.

Reply:
Any aversion to writing the one below to 65535 to reflect what is in the datasheet. This way one can search for 36863 and find it in the datasheet as well as 65535, whereas the hex values are nowhere to be found in the datasheet.

See Control Register (0x01) in the TSL2591 datasheet

Reply:
How about writing one as a comment and one as a constant?

Reply:
Put the comments in for the hex representation as well